apps/scan/frontend/src/screens/unconfigured_election_screen_wrapper.tsx

```diff
@@ -26,13 +22,20 @@ export function UnconfiguredElectionScreenWrapper(
   const configureMutation =
     configureFromElectionPackageOnUsbDrive.useMutation();
   // TODO move watching for USB drive to configure to the backend
-  useQueryChangeListener(usbDriveStatusQuery, {
-    onChange: (newUsbDriveStatus) => {
-      if (newUsbDriveStatus.status === 'mounted') {
-        configureMutation.mutate();
-      }
-    },
-  });
+  useEffect(() => {
+    if (
+      usbDriveStatusQuery.isSuccess &&
+      usbDriveStatusQuery.data.status === 'mounted'
+    ) {
+      configureMutation.mutate();
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Which dep is missed?

Reply:
It thinks configureMutation even though I include configureMutation.mutate and if you try to include the entire mutation a different lint rule tells you to just include the .mutate function.